SplitFiles/face.c

```diff
@@ -0,0 +1,182 @@
+#include <arch/zx.h>
+#include <stdio.h>
+#include "util.h"
+#include "face.h"
+
+void happyFace(const int x, const int y)
+{
+    int px = (x - 1) * 8;
+    int py = (y - 1) * 8;
+    *zx_pxy2saddr(px + 8, py) = 0x0;
+    *zx_pxy2saddr(px, py++) = 0x1;
```

Comment:
Why the left side _after_ the right?

Reply:
On the lines: no, in the sense that not all the lines are stacked one on top of the other aiui. Each line itself is linear but - from my understanding of the markdown on the z88dk tutorial they aren't necessarily stacked one on top of the other. But maybe I have misunderstood that?

On the order: Accident of how I coded it up with the unary ++ just.